paperqa/agents/search.py

```diff
@@ -405,42 +402,42 @@ async def process_file(
             logger.info(f"Complete ({title}).")
 
 
+# TODO: Use settings directly for some of this config
```

Comment:
What do you mean by use settings directly? You mean like move this to a method of `Settings`?